macros/get_organization_tag_columns.sql

```diff
@@ -0,0 +1,21 @@
+{% macro get_organization_tag_columns() %}
+
+{% set columns = [
+    {"name": "_fivetran_synced", "datatype": dbt_utils.type_timestamp()},
+    {"name": "organization_id", "datatype": dbt_utils.type_int()}
+] %}
+
+{% if target.type == 'redshift' %}
```

Comment:
@fivetran-jamie does this syntax align with the way you've approached keyword columns in the past?

Reply:
Updated to align with the syntax in the dbt_greenhouse_source package